conf/sources.json

```diff
@@ -5,7 +5,8 @@
 				"json_log",
 				"syslog_log",
 				"kv_log",
-				"csv_log"
+				"csv_log",
+				"osquery"
```

Comment:
this reads weird since 1 is a real life log source and the others are types. Maybe prefix the others with 'example_' to make it clear?